docs/en/observability/monitor-aws.asciidoc

```diff
@@ -57,16 +56,15 @@ image::creating-a-s3-bucket.png[S3 bucket creation]
 [[aws-step-two]]
 === Step 2:  Create an SQS Queue
 
-You should now have an S3 bucket in which you can export your logs, but you will
-also need an SQS queue. To avoid significant lagging with polling all
-log files from each S3 bucket, we will use Amazon Simple Queue Service (SQS).
-This will provide us with an Amazon S3 notification when a new S3 object is
-created. The {filebeat-ref}/filebeat-input-aws-s3.html[Filebeat S3 input]
-checks SQS for new messages regarding the new object created in S3 and uses the
-information in these messages to retrieve logs from S3 buckets. With this setup,
-periodic polling from each S3 bucket is not needed. Instead, the Filebeat S3
-input guarantees near real-time data collection from S3 buckets with both speed
-and reliability.
+You have created an S3 bucket. This will be where you can export your logs.
+This doesn't notify you when users create new S3 objects. Rather than poll
```

Comment:
**question:** @dedemorton : I had some trouble with this, because SQS was raised before it was defined or put in context. What do you think needs to be called out or clarified?

Reply:
I would remove the info about the S3 bucket because it's redundant with the previous section. Then I would explain what the SQS queue is and why you need to create one.  I think the sentence structure in your edit (3 sentences in a row that begin with "this") is a bit choppy. The reader has to figure out what "this" refers back to. I think you are moving in the right direction, but might need to change a few more things to make it work.

Reply:
I think this captures the point better, though perhaps not perfectly. 

```
Cloudwatch can export your logs to this new S3 bucket. Exporting the logs doesn't notify Filebeat that new logs are in the S3 bucket. Rather than poll each S3 bucket for these log files, you can create a message queue. You can use the Amazon Simple Queue Service (SQS). SQS creates and stores messages when logs reach the S3. The {filebeat-ref}/filebeat-input-aws-s3.html[Filebeat S3 input] checks SQS queue for S3 messages. Filebeat then uses the contents of these messages to retrieve logs from S3 buckets. The Filebeat S3 input guarantees reliable, near real-time data collection from S3 buckets.
```

Reply:
Yes that's much better. Sorry about my delayed response. Wasn't available to respond to GitHub last week. :-)